assets/src/apps/authoring/components/PropertyEditor/custom/CompJsonEditor.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { CSSProperties, useState } from 'react';
+interface JsonEditoProps {
+  jsonValue: any;
+  onChangeHandler: any;
```

Comment:
I think usually for the props it's meant to be an event, so just `onChange` here, the method that you pass in from the outside would be the "handler" of that event;

also I think the type is more known for the method
maybe
`onChange: (changedJson: any) => void;`